src/kata3.numberToReversedDigits.js

```diff
@@ -1,3 +1,9 @@
-const numberToReversedDigits = (number) => {};
+const numberToReversedDigits = (number) => {
+    return number.toString().split('').reverse().map(digit => parseInt(digit));
+};
+// convert number to string number.toString().split('').reverse()
+// convert strings to numbers
+// add to array
+// return array
```

Comment:
These comments should be removed as they arent adding any new information to the file